src/profile-logic/comparison.js

```diff
@@ -52,6 +52,7 @@ import type {
 import type { UrlState } from '../types/state';
```

Comment:
This commit is the result of changing the approach to add the `duration` array only when needed, from the other PR.